style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
```

Comment:
[OPTIONAL ] 
- [ ] It is best practice to access the markup document using the HTML class. 
`Hint: .josh would be nice than h1`